documentation/tools.md

```diff
@@ -52,7 +54,10 @@ Even though Cube Schema is a very lightweight vocabulary its structure needs to
 
 #### The structure of the observations
 
-When a cube provides an optional observation constraint through the [observationConstaint](#observationConstraint) property this can be tested as well
+When a cube provides an optional observation constraint through the [observationConstaint](#observationConstraint) property, this can be tested as well.
+
+The constraint should be a SHACL shape but it's not expected to have any [target declaration](https://www.w3.org/TR/shacl/#targets).
+The validation tool takes care of making all the observations a target for the constraint.
```

Comment:
@giacomociti good point 👍 

nit: I'd even suggest:
- it's expected to not have any [target declaration]

btw: this would also be worth mentioning in https://cube.link/#cubeconstraints - or would there be anything against that?